README.adoc

```diff
@@ -1723,6 +1723,28 @@ Time.zone.now # => Fri, 12 Mar 2014 22:04:47 EET +02:00
 Time.current # Same thing but shorter.
 ----
 
+=== Prefer `all_(day|week|month|quarter|year)` over range of date/time [[date-time-range]]
+
+Prefer `all_(day|week|month|quarter|year)` over `beginning_of_(day|week|month|quarter|year)..end_of_(day|week|month|quarter|year)`
+to get the range of date/time.
```

Comment:
"the range of date/time" -> "period"?
I understand that the current wording is technical, and the proposed is semantic.
Leaving it up to you what to go with.

Reply:
I chose `date/time` from the following documentation, but I think `period` will look like.
https://github.com/rails/rails/blob/v7.0.3/activesupport/lib/active_support/core_ext/date_and_time/calculations.rb#L299-L323

OTOH, this wording `date/time` is matched with the name of `Rails/DateTimeRange` cop.
https://github.com/rubocop/rubocop-rails/pull/730

It's always difficult to name 😅